libs/block-scripts/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "@local/tsconfig/node.json",
-  "include": ["bin.js", "scripts", "shared"]
+  "include": ["bin.js", "scripts", "shared"],
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
Even with `validateVersionedUrl` changed, `graph` is still throwing errors because of its dependency on `type-system`. I'm not sure if there's a better way of approaching this. We have `skipLibCheck: true` in the frontend already.

Reply:
Yes, it's the WASM stuff